node/node.go

```diff
@@ -89,13 +89,19 @@ import (
 )
 
 var (
-	genesisHashKey  = []byte("genesisID")
+	genesisHashKey = []byte("genesisID")
+	sessionKey     = []byte("session")
```

Comment:
We could also consider adding some `node` prefixdb to put keys under but I didn't do this since we already write genesisID directly.